corehq/apps/app_manager/xpath.py

```diff
@@ -180,9 +180,8 @@ class CaseTypeXpath(CaseSelectionXPath):
     selector = '@case_type'
 
     def case(self, instance_name='casedb', case_name='case'):
-        return CaseXPath(u"instance('{inst}')/{inst}/{case}[{sel}='{self}']".format(
-            inst=instance_name, case=case_name, sel=self.selector, self=self
-        ))
+        quoted = CaseTypeXpath(u"'{}'".format(self))
```

Comment:
Considered `quoted = CaseTypeXpath(XPath.string(self))`, but that felt like a slippery slope that ends at replacing every "/" with a call to `slash()`.